### PR TITLE
German Translations

### PR DESCRIPTION
**Pull request by jklippel**
Added german translations to translations file.

### CODE REVIEW

**Comment by spowelljr** [comment]
@jklippel 
```
--- FAIL: TestTranslationFilesValid (0.05s)
    --- FAIL: TestTranslationFilesValid/de.json (0.01s)
        translate_test.go:151: line "containers paused status: {{.paused}}": "Container in pausiert status: {{.pause}}" has mismatching variables
            original string variables: [{{.paused}}] do not match translated variables: [{{.pause}}]
        translate_test.go:151: line "{{.driver_name}} has only {{.container_limit}}MB memory but you specified {{.specified_memory}}MB": "{{.driver}} hat nur {{.container_limit}}MB Speicher aber spezifiziert wurden {{.specified_memory}}MB" has mismatching variables
            original string variables: [{{.container_limit}} {{.driver_name}} {{.specified_memory}}] do not match translated variables: [{{.container_limit}} {{.driver}} {{.specified_memory}}]
        translate_test.go:143: line "You have chosen to disable the CNI but the \\\"{{.name}}\\\" container runtime requires CNI": "Sie haben den CNI Treiber deaktiviert, aber die \\\"{{.name}\\\" Container Laufzeitumgebung benötigt ein CNI" has mismatching number of variables
            original string variables: [{{.name}}]; translated variables: []
        translate_test.go:143: line "Verifying {{.addon_name}} addon...": "Verifiziere {.addon_name}} Addon..." has mismatching number of variables
            original string variables: [{{.addon_name}}]; translated variables: []
        translate_test.go:151: line "Requested memory allocation {{.requested}}MiB is less than the usable minimum of {{.minimum_memory}}MB": "Die angeforderte Speicherzuweisung {{.request}}MB ist weniger als das verwendbare Minimum {{.minimum_memory}}MB" has mismatching variables
            original string variables: [{{.minimum_memory}} {{.requested}}] do not match translated variables: [{{.minimum_memory}} {{.request}}]
        translate_test.go:143: line "dashboard service is not running: {{.error}}": "Dashboard Service läuft nicht" has mismatching number of variables
            original string variables: [{{.error}}]; translated variables: []
        translate_test.go:151: line "Requested memory allocation ({{.requested}}MB) is less than the recommended minimum {{.recommend}}MB. Deployments may fail.": "Die angeforderte Speicherzuweisung {{.requested_size}} liegt unter dem zulässigen Mindestwert von {{.minimum_size}}MB. Deployments könnten fehlschlagen." has mismatching variables
            original string variables: [{{.recommend}} {{.requested}}] do not match translated variables: [{{.minimum_size}} {{.requested_size}}]
```

**Comment by jklippel** [comment]
@spowelljr Fixed

File: translations/de.json [hunk 1]
```diff
@@ -1,996 +1,1003 @@
...
+	"To disable this notice, run: 'minikube config set WantUpdateNotification false'\\n": "Um diesen Hinweis zu deaktivieren, starte: 'minikube config set WantUpdateNotification false'\\n",
+	"To disable update notices in general, run: 'minikube config set WantUpdateNotification false'\\n": "Um Hinweise generell zu deaktivieren, starte: 'minikube config set WantUpdateNotification false'\\n",
+	"To pull new external images, you may need to configure a proxy: https://minikube.sigs.k8s.io/docs/reference/networking/proxy/": "Um neue externe Images zu ziehen, müsste eventuell ein Proxy konfiguriert werden: https://minikube.sigs.k8s.io/docs/reference/networking/proxy/",
+	"To see addons list for other profiles use: `minikube addons -p name list`": "Um die Addon-List für andere Profile anzusehen, verwende: `minikube addons -p name list`",
+	"To set your Google Cloud project,  run:\n\n\t\tgcloud config set project \u003cproject name\u003e\n\nor set the GOOGLE_CLOUD_PROJECT environment variable.": "Um das Google Cloud project zu setzten,  starte:\n\n\t\tgcloud config set project <project name>\n\noder setze die Umgebungsvariabel GOOGLE_CLOUD_PROJECT.",
```
Comment (spowelljr) [inline translations/de.json]:
```suggestion
	"To set your Google Cloud project,  run:\n\n\t\tgcloud config set project \u003cproject name\u003e\n\nor set the GOOGLE_CLOUD_PROJECT environment variable.": "Um das Google Cloud project zu setzten,  starte:\n\n\t\tgcloud config set project \u003cproject name\u003e\n\noder setze die Umgebungsvariabel GOOGLE_CLOUD_PROJECT.",
```
For conformity we should keep `<>` as unicode.

File: translations/de.json [hunk 2]
```diff
@@ -1,996 +1,1003 @@
...
 	"127.0.0.1": "",
 	"==\u003e Audit \u003c==": "",
-	"==\u003e Last Start \u003c==": "",
-	"A VPN or firewall is interfering with HTTP access to the minikube VM. Alternatively, try a different VM driver: https://minikube.sigs.k8s.io/docs/start/": "",
-	"A firewall is blocking Docker the minikube VM from reaching the image repository. You may need to select --image-repository, or use a proxy.": "",
-	"A firewall is interfering with minikube's ability to make outgoing HTTPS requests. You may need to change the value of the HTTPS_PROXY environment variable.": "",
-	"A firewall is likely blocking minikube from reaching the internet. You may need to configure minikube to use a proxy.": "",
-	"A set of apiserver IP Addresses which are used in the generated certificate for kubernetes.  This can be used if you want to make the apiserver available from outside the machine": "",
+	"==\u003e Last Start \u003c==": "==> Letzter Start <==",
```
Comment (spowelljr) [inline translations/de.json]:
```suggestion
	"==\u003e Last Start \u003c==": "==\u003e Letzter Start \u003c==",
```

File: translations/de.json [hunk 3]
```diff
@@ -1,996 +1,1003 @@
...
+	"Choose a smaller value for --memory, such as 2000": "Wählen Sie einen schmaleren Wert für --memory (z.B. 2000)",
+	"ChromeOS is missing the kernel support necessary for running Kubernetes": "ChromeOS besitzt nicht die notwendige Kernel-Unterstützung um Kubernetes auszuführen",
+	"Cluster was created without any CNI, adding a node to it might cause broken networking.": "Der Cluster wurde ohne CNI erstellt, das Hinzufügen eines Nodes kann zu einem kaputten Netzwerk-Setup führen",
+	"Configuration and Management Commands:": "Konfigurations- und Management-Befehle:",
+	"Configure a default route on this Linux host, or use another --driver that does not require it": "Konfigurieren Sie eine Default-Route auf diesem Linux Host oder verwenden Sie einen anderen --driver, die dies nicht benötigt",
+	"Configure an external network switch following the official documentation, then add `--hyperv-virtual-switch=\u003cswitch-name\u003e` to `minikube start`": "Konfigurieren Sie einen externen Netzwerk-Switch mit Hilfe der offiziellen Dokumentation, dann fügen Sie `--hyperv-virtual-switch=<switch-name>` zum Start-Befehl `minikube start` hinzu",
```
Comment (spowelljr) [inline translations/de.json]:
```suggestion
	"Configure an external network switch following the official documentation, then add `--hyperv-virtual-switch=\u003cswitch-name\u003e` to `minikube start`": "Konfigurieren Sie einen externen Netzwerk-Switch mit Hilfe der offiziellen Dokumentation, dann fügen Sie `--hyperv-virtual-switch=\u003cswitch-name\u003e` zum Start-Befehl `minikube start` hinzu",
```

File: translations/de.json [hunk 4]
```diff
@@ -1,996 +1,1003 @@
...
-	"Docker Desktop has less than 2 CPUs configured, but Kubernetes requires at least 2 to be available": "",
-	"Docker Desktop is configured for Windows containers, but Linux containers are required for minikube": "",
-	"Docker Desktop only has {{.size}}MiB available, less than the required {{.req}}MiB for Kubernetes": "",
-	"Docker Desktop only has {{.size}}MiB available, you may encounter application deployment failures.": "",
-	"Docker container exited prematurely after it was created, consider investigating Docker's performance/health.": "",
-	"Docker has less than 2 CPUs available, but Kubernetes requires at least 2 to be available": "",
-	"Docker inside the VM is unavailable. Try running 'minikube delete' to reset the VM.": "",
-	"Docs have been saved at - {{.path}}": "",
-	"Documentation: {{.url}}": "",
+	"Disk size allocated to the minikube VM (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g).": "Größe des der minikube-VM zugewiesenen Festplatte (Format: <Nummer> [<Einheit>], wobei Einheit = b, k, m oder g).",
```
Comment (spowelljr) [inline translations/de.json]:
```suggestion
	"Disk size allocated to the minikube VM (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g).": "Größe des der minikube-VM zugewiesenen Festplatte (Format: \u003cNummer\u003e [\u003cEinheit\u003e], wobei Einheit = b, k, m oder g).",
```

File: translations/de.json [hunk 5]
```diff
@@ -1,996 +1,1003 @@
...
+	"Another minikube instance is downloading dependencies... ": "Eine andere Minikube-Instanz lädt Abhängigkeiten herunter... ",
+	"Another program is using a file required by minikube. If you are using Hyper-V, try stopping the minikube VM from within the Hyper-V manager": "Ein anderes Programm benutzt eine Datei, die Minikube benötigt. Wenn Sie Hyper-V verwenden, versuchen Sie die minikube VM aus dem Hyper-V Manager heraus zu stoppen",
+	"At least needs control plane nodes to enable addon": "Benötige mindestens Control Plane Nodes um das Addon zu aktivieren",
+	"Automatically selected the {{.driver}} driver": "Treiber {{.driver}} wurde automatisch ausgewählt",
+	"Automatically selected the {{.driver}} driver. Other choices: {{.alternates}}": "Treiber {{.driver}} wurde automatisch ausgewählt. Andere Möglichkeiten: {{.alternates}}",
+	"Available Commands": "Verfügbare Befehle",
+	"Basic Commands:": "Grundlegende Befehle",
```
Comment (spowelljr) [inline translations/de.json]:
```suggestion
	"Basic Commands:": "Grundlegende Befehle:",
```

File: translations/de.json [hunk 6]
```diff
@@ -1,996 +1,1003 @@
...
+	"Deletes a local Kubernetes cluster": "Löscht einen lokalen Kubernetes Cluster",
+	"Deletes a local Kubernetes cluster. This command deletes the VM, and removes all\nassociated files.": "Löscht einen lokalen Kubernetes Cluster. Dieser Befehl löscht die VM und entfernt alle\nzugehörigen Dateien.",
 	"Deletes a local kubernetes cluster. This command deletes the VM, and removes all associated files.": "Damit wird ein lokaler Kubernetes-Cluster gelöscht. Mit diesem Befehl wird die VM entfernt und alle zugehörigen Dateien gelöscht.",
-	"Deletes a node from a cluster.": "",
+	"Deletes a node from a cluster.": "Löscht einen Node aus einem Cluster.",
 	"Deleting \"{{.profile_name}}\" in {{.driver_name}} ...": "{{.profile_name}}\" in {{.driver_name}} wird gelöscht...",
-	"Deleting container \"{{.name}}\" ...": "",
-	"Deleting existing cluster {{.name}} with different driver {{.driver_name}} due to --delete-on-failure flag set by the user. ": "",
-	"Deleting node {{.name}} from cluster {{.cluster}}": "",
+	"Deleting container \"{{.name}}\" ...": "Lösche Container \"{{.name}}\"",
```
Comment (spowelljr) [inline translations/de.json]:
```suggestion
	"Deleting container \"{{.name}}\" ...": "Lösche Container \"{{.name}}\" ...",
```

File: translations/de.json [hunk 7]
```diff
@@ -1,996 +1,1003 @@
...
-	"ERROR creating `registry-creds-acr` secret": "",
-	"ERROR creating `registry-creds-dpr` secret": "",
-	"ERROR creating `registry-creds-ecr` secret: {{.error}}": "",
-	"ERROR creating `registry-creds-gcr` secret: {{.error}}": "",
-	"Either systemctl is not installed, or Docker is broken. Run 'sudo systemctl start docker' and 'journalctl -u docker'": "",
-	"Enable addons. see `minikube addons list` for a list of valid addon names.": "",
+	"Downloading Kubernetes {{.version}} preload ...": "Lade Kubernetes {{.version}} herunter ...",
+	"Downloading VM boot image ...": "Lade VM boot image herunter ...",
+	"Downloading driver {{.driver}}:": "Lade Treiber {{.driver}} herunter:",
+	"Duration until minikube certificate expiration, defaults to three years (26280h).": "Dauer bis das Minikube-Zertifikat abläuft, Default ist drei Jahre (26280 Stunden)",
```
Comment (spowelljr) [inline translations/de.json]:
```suggestion
	"Duration until minikube certificate expiration, defaults to three years (26280h).": "Dauer bis das Minikube-Zertifikat abläuft, Default ist drei Jahre (26280 Stunden).",
```

File: translations/de.json [hunk 8]
```diff
@@ -1,996 +1,1003 @@
...
-	"Configuring {{.name}} (Container Networking Interface) ...": "",
-	"Confirm that you have a working internet connection and that your VM has not run out of resources by using: 'minikube logs'": "",
-	"Confirm that you have supplied the correct value to --hyperv-virtual-switch using the 'Get-VMSwitch' command": "",
-	"Connect to LoadBalancer services": "",
-	"Consider creating a cluster with larger memory size using `minikube start --memory SIZE_MB` ": "",
-	"Consider increasing Docker Desktop's memory size.": "",
-	"Continuously listing/getting the status with optional interval duration.": "",
-	"Control Plane could not update, try minikube delete --all --purge": "",
-	"Copy the specified file into minikube": "",
+	"Booting up control plane ...": "Starte Control-Plane",
```
Comment (spowelljr) [inline translations/de.json]:
```suggestion
	"Booting up control plane ...": "Starte Control-Plane ...",
```

**Review by spowelljr — CHANGES_REQUESTED** [review]

**Comment by jklippel** [comment]
@spowelljr Thank you for your review. I applied the requested changes (IDE autocorrecting unicode away :/). I had to do the changes manually, I hope I did not miss anything. For some strange reason github showed all your comments as outdated and I could not collect your suggestions into a batch to commit.

File: translations/de.json [hunk 9]
```diff
@@ -1,996 +1,1003 @@
...
+	"Failed to push images": "Remote-Aktualisierung (push) des Images fehlgeschlagen",
+	"Failed to read temp": "Lesen von temp fehlgeschlagen",
+	"Failed to reload cached images": "Erneutes Laden der gecachten Images fehlgeschlagen",
+	"Failed to remove image": "Entfernen des Images fehlgeschlagen",
+	"Failed to remove images for profile {{.pName}} {{.error}}": "Entfernen des Images für Profil {{.pName}} fehlgeschlagen {{.error}}",
+	"Failed to save config {{.profile}}": "Speichern der Konfiguration {{.profile}} fehlgeschlagen",
+	"Failed to save dir": "Speichern des Verzeichnisses fehlgeschlagen",
+	"Failed to save image": "Speichern des Images fehlgeschlagen",
+	"Failed to save stdin": "Speichern der Standard-Eingabe fehlgeschlagen",
+	"Failed to set NO_PROXY Env. Please use `export NO_PROXY=$NO_PROXY,{{.ip}}": "NO_PROXY Env konnte nicht festgelegt werden. Benutzen Sie `export NO_PROXY=$NO_PROXY, {{.ip}}",
```
Comment (spowelljr) [inline translations/de.json]:
```suggestion
	"Failed to set NO_PROXY Env. Please use `export NO_PROXY=$NO_PROXY,{{.ip}}": "NO_PROXY Env konnte nicht festgelegt werden. Benutzen Sie `export NO_PROXY=$NO_PROXY,{{.ip}}",
```
Need no space after the comma as well

File: translations/de.json [hunk 10]
```diff
@@ -1,996 +1,1003 @@
...
+	"Failed to set NO_PROXY Env. Please use `export NO_PROXY=$NO_PROXY,{{.ip}}`.": "NO_PROXY Env konnte nicht festgelegt werden. Benutzen Sie `export NO_PROXY=$NO_PROXY, {{.ip}}`.",
```
Comment (spowelljr) [inline translations/de.json]:
```suggestion
	"Failed to set NO_PROXY Env. Please use `export NO_PROXY=$NO_PROXY,{{.ip}}`.": "NO_PROXY Env konnte nicht festgelegt werden. Benutzen Sie `export NO_PROXY=$NO_PROXY,{{.ip}}`.",
```

File: translations/de.json [hunk 11]
```diff
@@ -1,996 +1,1003 @@
...
-	"Please re-eval your docker-env, To ensure your environment variables have updated ports:\n\n\t'minikube -p {{.profile_name}} docker-env'\n\n\t": "",
-	"Please re-eval your podman-env, To ensure your environment variables have updated ports:\n\n\t'minikube -p {{.profile_name}} podman-env'\n\n\t": "",
-	"Please run `minikube logs --file=logs.txt` and attach logs.txt to the GitHub issue.": "",
-	"Please see {{.documentation_url}} for more details": "",
-	"Please specify the directory to be mounted: \n\tminikube mount \u003csource directory\u003e:\u003ctarget directory\u003e   (example: \"/host-home:/vm-home\")": "",
-	"Please specify the path to copy: \n\tminikube cp \u003csource file path\u003e \u003ctarget file absolute path\u003e (example: \"minikube cp a/b.txt /copied.txt\")": "",
-	"Please try purging minikube using `minikube delete --all --purge`": "",
+	"Paused {{.count}} containers": "{{.count}} Container pausiert",
+	"Paused {{.count}} containers in: {{.namespaces}}": "{{.count}} Container pausiert in: {{.namespaces}}",
+	"Pausing node {{.name}} ... ": "Pausiere Node {{.name}}",
```
Comment (spowelljr) [inline translations/de.json]:
```suggestion
	"Pausing node {{.name}} ... ": "Pausiere Node {{.name}} ...",
```

**Review by spowelljr — CHANGES_REQUESTED** [review]

**Comment by spowelljr** [comment]
@jklippel thanks for taking your time to do the translations, much appreciated! I see that, I'm not sure why that's happening. I just added three more comments and after that it'll be good to merge!

**Comment by jklippel** [comment]
@spowelljr Thanks again. I added the additional proposed changes as well.

**Review by spowelljr — APPROVED** [review]
Looks good, thank you very much!

**Pull request merged**